paddle/framework/block_desc.cc

```diff
@@ -18,7 +18,7 @@ limitations under the License. */
 namespace paddle {
 namespace framework {
 
-VarDescBind *BlockDescBind::NewVar(const std::string &name) {
+VarDescBind *BlockDescBind::GetOrCreateVar(const std::string &name) {
```

Comment:
this should not me changed.

Reply:
It's the same with `Scope`, also do the same logic of **find or create**.